client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/presentation/SessionActivity.java

```diff
@@ -351,8 +358,12 @@ protected void onDestroy() {
         // remove clipboard listener
         mClipboardManager.removeClipboardboardChangedListener(this);
 
+        // FIXME: prevent crashing app in DeX Mode: app crashes with popup window when user manually closes the Window
+        DexFixCloseWin();
```

Comment:
Hack (along with the memory leak mentioned above)
From the description this is caused by not stopping the running session and then freeing up the resources still in use by the native code. Remove